plugins/net_plugin/net_plugin.cpp

```diff
@@ -2728,8 +2728,8 @@ namespace eosio {
       chain_lib_id = cc.last_irreversible_block_id();
       chain_head_blk_num = cc.head_block_num();
       chain_head_blk_id = cc.head_block_id();
-      chain_fork_head_blk_num = cc.fork_db_pending_head_block_num();
-      chain_fork_head_blk_id = cc.fork_db_pending_head_block_id();
+      chain_fork_head_blk_num = cc.fork_db_head_block_num();
+      chain_fork_head_blk_id = cc.fork_db_head_block_id();
```

Comment:
in non-irreversible mode, we will store the same in `chain_head_blk_num` and `chain_fork_head_blk_num`. Same for id. Is this what we want?

Reply:
Thanks. `chain_head_*` is not used.